src/js/object.js

```diff
@@ -0,0 +1,27 @@
+const BUTTON = Object.freeze({
+    EDIT: "menu-edit-button",
+    REMOVE: "menu-remove-button",
+    SOLDOUT: "menu-sold-out-button"
+});
+
+const CATEGORYNAME = Object.freeze({
+    espresso: "☕ 에스프레소",
+    frappuccino: "🥤 프라푸치노",
+    blended: "🍹 블렌디드",
+    teavana: "🫖 티바나",
+    desert: "🍰 디저트"
+});
+
+let info = {
+    _category: "espresso",
+
+    get category() { return this._category; }, 
+    set category(value) { this._category = value; },
+}
+
+class menuInfo {
+    constructor(menuName, soldoutClass="") {
+        this.menuName = menuName;
+        this.soldoutClass = soldoutClass;
+    }
+}
```

Comment:
```suggestion
export const BUTTON = Object.freeze({
    EDIT: "menu-edit-button",
    REMOVE: "menu-remove-button",
    SOLDOUT: "menu-sold-out-button"
});

export const CATEGORYNAME = Object.freeze({
    espresso: "☕ 에스프레소",
    frappuccino: "🥤 프라푸치노",
    blended: "🍹 블렌디드",
    teavana: "🫖 티바나",
    desert: "🍰 디저트"
});

export let info = {
    _category: "espresso",

    get category() { return this._category; }, 
    set category(value) { this._category = value; },
}

export class menuInfo {
    constructor(menuName, soldoutClass="") {
        this.menuName = menuName;
        this.soldoutClass = soldoutClass;
    }
}
```

html에 직접 script를 불러오는 대신 모듈 내보내기를 쓰면 좀 더 쉽게 관리할 수 있어요!
참조 링크: https://ko.javascript.info/import-export

불러올때는 import를 써주시면 됩니다
```js
improt { BUTTON } from './object.js';
```
만약 export default로 내보냈으면 괄호를 생략하고 가져올 수 있어요
```js
export default class menuInfo {
    constructor(menuName, soldoutClass="") {
        this.menuName = menuName;
        this.soldoutClass = soldoutClass;
    }
}

improt menuInfo from './object.js';
```